presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerClusterStateProvider.java

```diff
@@ -337,19 +333,6 @@ public Map<String, MemoryInfo> getWorkerMemoryInfo()
         }, e -> e.getValue().getNodeStatus().getMemoryInfo()));
     }
 
-    private void validateCoordinatorConsistency()
```

Comment:
How will we differentiate between a true failure from heartbeats not arriving and a temporary failure which this PR tries to address.?

Reply:
As of today we don't have the visibility into the failure coming from true failure or due to this validation. But during the investigation we found in prod clusters that the failures are due to this validation as the cluster was having stale coordinator information. So removing this check will make sure going forward the failures will happen for a real reason and not this validation. Hope that answers your question.